File: Java/UberX.java
class UberX extends Uber_cars {
    String marca;
    String model;

    public UberX(String license, Cuenta driver, String marca, String model){
        super(license, driver);
        this.marca = marca;
        this.model = model;
    }
    
    @Override
    void printDataCars() {
        super.printDataCars();
        System.out.println("Marca" + marca + "modelo" + model);
    }
}

File: PHP/UberX.php
<?php
require_once('uber_cars.php');
class UberX extends uber_cars {
    public $marca;
    public $modelo;

    public function __construct($license, $driver, $marca, $modelo) {
        parent:: __construct($license, $driver);
        $this->marca = $marca;
        $this->modelo = $modelo;
    }

    public function PrintDataCar(){
        parent::PrintDataCar();
        echo"
        Modelo: $this->modelo
        Marca: $this->marca
        ";
    }
}
?>